src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -220,53 +224,101 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
   }
 
   copyFile(rightClickedFile: any) {
-    if(!this.fileToCopy){
+    if(this.fileToCopyOrCut == ''){
+      this.rightClickPropertiesFolder.push(
+        { text: "Paste", action:() => { 
+          this.pasteFile(this.fileToCopyOrCut,this.rightClickedFile.path)
+        }}
+      );
+      this.rightClickPropertiesPanel.push(
+        { text: "Paste", action:() => { 
+          this.pasteFile(this.fileToCopyOrCut,this.path)
+        }}
+      );
+    }
+    this.fileToCopyOrCut = rightClickedFile.path
+    this.isCutOperation = false;
+  }
+
+  cutFile(rightClickedFile: any) {
+    if(this.fileToCopyOrCut == ''){
       this.rightClickPropertiesFolder.push(
         { text: "Paste", action:() => { 
-          this.pasteFile(this.fileToCopy,this.rightClickedFile)
+          this.pasteFile(this.fileToCopyOrCut,this.rightClickedFile.path)
```

Comment:
Why not have paste file 3rd argument being whether it is cut or copy?

Reply:
Done